### PR TITLE
Feature/alaska prod enable es

### PR DESCRIPTION
**Pull request by dougszumski**

### CODE REVIEW

File: etc/kayobe/kolla/inventory/overcloud-components.j2 [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# This inventory section provides a mapping of the components to top level
```
Comment (markgoddard) [inline etc/kayobe/kolla/inventory/overcloud-components.j2:1]: Perhaps you could outline why we're using a custom inventory here, or alternatively add a comment next to lines that are changed from the original.

Comment (dougszumski) [reply etc/kayobe/kolla/inventory/overcloud-components.j2:1]: Will do. I put a diff in the commit message to check nothing unexpected happened.

Comment (markgoddard) [reply etc/kayobe/kolla/inventory/overcloud-components.j2:1]: Oh, yes I see now. I should make more effort to look at the individual commits, especially when you've gone to the effort of separating them nicely.

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/kolla/inventory/overcloud-components.j2 (hunk 1 above)
Comment (dougszumski) [reply etc/kayobe/kolla/inventory/overcloud-components.j2:1]: No worries - github seems good at hiding the commit messages.

**Pull request merged**